types/enginko/MCF-LW12CO2E/uplink.js

```diff
@@ -783,16 +825,13 @@ function parseReportData(payload) {
   } else {
     return null;
   }
-  return [content, topic];
-}
-
-function isNumeric(str) {
-  return /^\-?[0-9]+(e[0-9]+)?(\.[0-9]+)?$/.test(str);
+  return content;
 }
 
 function consume(event) {
   const payload = event.data.payloadHex;
   const sample = {};
+  const sample2 = {};
```

Comment:
why do you need to resulting samples?

Reply:
The sensor can send two samples with the same structure at the same time. The only difference being that they're 10 minutes apart. Thats why i need two.

Reply:
please try to make this more generic, something along the lines of 
```
emit("sample", parseSample(event.data.payloadHex[0:50]));
emit("sample", parseSample(event.data.payloadHex[50:79]));
```